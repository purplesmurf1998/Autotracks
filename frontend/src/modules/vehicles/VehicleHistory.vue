<template>
  <div>
    <h1>Vehicle History</h1>
  </div>
</template>

<script>
export default {
  name: 'VehicleHistory',
  props: ["vehicle"],
};
</script>

<style>
</style>