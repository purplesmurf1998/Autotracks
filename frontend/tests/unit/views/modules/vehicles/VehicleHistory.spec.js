import {createLocalVue, shallowMount} from '@vue/test-utils'
import CoreuiVue from '@coreui/vue-pro'
import Vuex from 'vuex'

import VehicleHistory from './../../../../../src/modules/vehicles/VehicleHistory.vue'

const localVue = createLocalVue()
localVue.use(CoreuiVue)
localVue.use(Vuex)

const store = new Vuex.Store({
    state: {
      sidebarShow: 'responsive',
      sidebarMinimize: false,
      asideShow: false,
      darkMode: false
    }
  })

  describe('VehicleHistory.vue', () => {
    const wrapper = shallowMount(VehicleHistory, { 
      store, 
      localVue
    })
    it('has a name', () => {
      expect(VehicleHistory.name).toBe('VehicleHistory')
    })
    it('should render correct content', () => {
        expect(wrapper.find('h1').text()).toMatch('Vehicle History')
    })
    test('renders correctly', () => {
      expect(wrapper.element).toMatchSnapshot()
    })
  })