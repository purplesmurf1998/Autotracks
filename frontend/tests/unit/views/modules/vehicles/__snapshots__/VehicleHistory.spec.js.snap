// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VehicleHistory.vue renders correctly 1`] = `
<div>
  <h1>
    Vehicle History
  </h1>
</div>
`;
